beacon-chain/core/helpers/validators.go

```diff
@@ -357,10 +358,11 @@ func BeaconProposerIndexAtSlot(ctx context.Context, state state.ReadOnlyBeaconSt
 //	      candidate_index = indices[compute_shuffled_index(i % total, total, seed)]
 //	      random_byte = hash(seed + uint_to_bytes(uint64(i // 32)))[i % 32]
 //	      effective_balance = state.validators[candidate_index].effective_balance
-//	      if effective_balance * MAX_RANDOM_BYTE >= MAX_EFFECTIVE_BALANCE * random_byte:
```

Comment:
I think it's fine to not remove this line for pseudocode since implementation still takes `maxEB = params.BeaconConfig().MaxEffectiveBalance` anyway

Reply:
It's not removed, it's modified 1 line down